experiment/measurer/test_measure_manager.py

```diff
@@ -25,7 +25,8 @@
 from database import models
 from database import utils as db_utils
 from experiment.build import build_utils
-from experiment.measurer import measure_manager
+from experiment.measurer import measure_manager, experiment_coverage_utils as \
```

Comment:
Please don't import two modules on one line (think it's a style-guide violation).
Also, please just import experiment_coverage_utils as experiment_coverage_utils (not exp_cov_utils please).